src/adapter/src/catalog/builtin.rs

```diff
@@ -2132,6 +2132,22 @@ FROM
 GROUP BY object_id, collection_timestamp",
 });
 
+pub const MZ_OBJECT_OID_ALIAS: BuiltinView = BuiltinView {
+    name: "mz_object_oid_alias",
+    schema: MZ_CATALOG_SCHEMA,
```

Comment:
Can this go in `mz_internal` instead? It's pretty in the weeds and would love to avoid it being part of the stable public API.

Reply:
Done